### DISCUSSION TITLE
Configuration to kafka connect with TLS not working with p12 trustore and keystore ?

### DISCUSSION THREAD

**Discussion by hervekhg** [tchiotludo/akhq#1020] [Q&A] [answered]
Hello,
I'm trying to connect AKHQ to Kafka-connect using TLS. When I try to use keystore and trustore on p12 format (instead of jks) it's not working. The jks is mandatory ? We would like to use p12.

Below our setup with p12
```bash
connect:
        - name: plusc
          url:  "https://scheam-plusc:8083"
          ssl-trust-store: "/tmp/truststore.p12"
          ssl-trust-store-password: "xxxxxx"
          ssl-key-store: "/tmp/keystore.p12"
          ssl-key-store-password: "xxxxxx"
```

**Comment by tchiotludo** [comment, marked as answer]
As I see, jks was mandatory. Internally, we use [Source Lab Kafka Connect client](https://github.com/SourceLabOrg/kafka-connect-client) and there is no option to choose keystore format, the format is hardcoded [here](https://github.com/SourceLabOrg/kafka-connect-client/blob/1492a269127e7fbc16435de4a20bfdaf56a87bde/src/main/java/org/sourcelab/kafka/connect/apiclient/rest/HttpsContextBuilder.java#L136). 

You will need to create an issue or a PR on this project if you want to use alternative type of trustore
